tests/integration/security/testdata/authz/v1beta1-negative-match.yaml.tmpl

```diff
@@ -22,38 +22,57 @@ spec:
 apiVersion: security.istio.io/v1beta1
 kind: AuthorizationPolicy
 metadata:
-  name: policy-c-same-namespace
+  name: policy-{{ .dst1 }}-same-namespace
   namespace: "{{ .Namespace }}"
 spec:
   selector:
     matchLabels:
-      "app": "c"
+      "app": "{{ .dst1 }}"
   action: DENY
   rules:
   - from:
     - source:
         notNamespaces: ["{{ .Namespace }}"]
 ---
 
-# The following policy denies access to workload d if it's not mTLS, in other words,
-# it allows only mTLS traffic to access workload d
+# The following policy denies access to a workload if it's not mTLS, in other words,
+# it allows only mTLS traffic to access the workload
 
 apiVersion: security.istio.io/v1beta1
 kind: AuthorizationPolicy
 metadata:
-  name: policy-d-mtls-traffic
+  name: policy-{{ .dst2 }}-mtls-traffic
   namespace: "{{ .Namespace }}"
 spec:
   selector:
     matchLabels:
-      "app": "d"
+      "app": "{{ .dst2 }}"
   action: DENY
   rules:
   - from:
     - source:
         notPrincipals: ["*"]
 ---
 
+# The following policy denies access to path with prefix "/prefix" except "/prefix/allowlist" to workload
```

Comment:
can you also add a TODO that the following policy is a duplicate of `policy-{{ .dst0 }}-deny` and can be removed once the test framework supports multiple vm workloads?

Reply:
Thanks, done.